packages/desktop-client/src/components/util/AmountInput.tsx

```diff
@@ -136,7 +137,9 @@ export function AmountInput({
       <Button
         variant="bare"
         isDisabled={disabled}
-        aria-label={`Make ${symbol === '-' ? 'positive' : 'negative'}`}
+        aria-label={t('Make {{type}}', {
+          type: symbol === '-' ? t('positive') : t('negative'),
+        })}
```

Comment:
```suggestion
        aria-label={symbol === '-' ? t('Make positive') : t('Make negative')}
```
(so we don't concatenate translations)

Reply:
good shout: 586012f47098568620fa7ff754cbc0fa574a815c